build-tools/android-toolchain/android-toolchain.targets

```diff
@@ -36,7 +36,7 @@
   <Target Name="_UnzipFiles"
       DependsOnTargets="_DetermineItems"
       Inputs="@(_PlatformAndroidSdkItem->'$(AndroidToolchainCacheDirectory)\%(Identity)')"
-      Outputs="$(AndroidToolchainDirectory)\.stamp-sdk">
+      Outputs="@(_PlatformAndroidSdkItem->'$(AndroidToolchainDirectory)\sdk\%(DestDir)\.stamp-sdk') $(AndroidToolchainDirectory)\ndk\.stamp-ndk">
```

Comment:
Not everything sets `%(AndroidSdkItem.DestDir)`, and thus `%(DestDir)` may be empty .

Additionally, it's not necessarily easy to detect what `%(DestDir)` could be; there could be *multiple* "top level" directories extracted from a single .zip. (For the most part, our extraction code doesn't care.)

I would thus suggest using `%(Identity)` within the `.stamp` filename:

    @(_PlatformAndroidSdkItem->'$(AndroidToolchainDirectory)\sdk\.stamp-%(Identity)')

Finally, you need to use `;`, not ` `, to separate file paths. Thus:

    Outputs="@(_PlatformAndroidSdkItem->'$(AndroidToolchainDirectory)\sdk\.stamp-%(Identity)');$(AndroidToolchainDirectory)\ndk\.stamp-ndk">